docs/hub-tutorials/join-mainnet.md

```diff
@@ -261,12 +261,21 @@ For more information on performing the manual upgrades, see [Releases & Upgrades
 
 It is possible to sync from previous versions of the Cosmos Hub. See the matrix below for the correct `gaia` version. See the [mainnet archive](https://github.com/cosmos/mainnet) for historical genesis files.
 
-| Chain Id      | Gaia Version  |
-| -----------   | -------- |
-| `cosmoshub-4` | `v4.2.1` |
-| `cosmoshub-3` | `v2.0.x` |
-| `cosmoshub-2` | `v1.0.x` |
-| `cosmoshub-1` | `v0.0.x` |
+
+| Upgrade Name        | Date          | Height    | Chain Identifier | Tm      | Cosmos SDK | Gaia                     | IBC                      |
+|---------------------|---------------|-----------|---------------|------------|------------|--------------------------|--------------------------|
+| Mainnet Launch      | 13/03/19    | 0         | `cosmoshub-1` | [v0.31.x](https://github.com/tendermint/tendermint/releases/tag/v0.31.11)          | [v0.33.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.33.2)    |  _Included in Cosmos SDK_ | n/a                      |
+| Upgrade #1          | 21/01/20    | 500,043   | `cosmoshub-2` | [v0.31.x](https://github.com/tendermint/tendermint/releases/tag/v0.31.11)          | [v0.34.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.34.10)    |   _Included in Cosmos SDK_)                  | n/a                      |
+| Upgrade #2          | 07/08/20    | 2,902,000 | `cosmoshub-3` | [v0.32.x](https://github.com/tendermint/tendermint/releases/tag/v0.32.14)          | [v0.37.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.37.15)    | [v2.0.x](https://github.com/cosmos/gaia/releases/tag/v2.0.14)                   | n/a                      |
+| Stargate            | 18/02/21    | 5,200,791 | `cosmoshub-4` | [v0.34.x](https://github.com/tendermint/tendermint/releases/tag/v0.34.3)          | [v0.40.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.1)    | [v4.0.x](https://github.com/cosmos/gaia/releases/tag/v4.0.6)                   | _Included in Cosmos SDK_ |
+| Security Hard Fork  | ?             | ?         | `cosmoshub-4` | [v0.34.x](https://github.com/tendermint/tendermint/releases/tag/v0.34.8)          | [v0.41.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.41.4)    | [v4.2.x](https://github.com/cosmos/gaia/releases/tag/v4.2.1)                   | _Included in Cosmos SDK_ |
```

Comment:
Should we leave this as TODO?

Reply:
Looks like the block `482100` https://forum.cosmos.network/t/critical-cosmossdk-security-advisory-updated/2211

Reply:
looks like this was a missing rolling hardfork after mainnet launch and before upgrade #1. Added but still missing info from the 2021 hardfork. only think i could find about it was gaia release...